target/linux/ar71xx/config-4.4

```diff
@@ -47,6 +47,7 @@ CONFIG_ATH79_MACH_AP136=y
 CONFIG_ATH79_MACH_AP143=y
 CONFIG_ATH79_MACH_AP147=y
 CONFIG_ATH79_MACH_AP152=y
+CONFIG_ATH79_MACH_AP531B0=y
```

Comment:
> sorry, what`s file attributes ?

What system did you make your changes on, was it Windows?
Have you build and tested your changes at all?

[File system permissions](https://en.wikipedia.org/wiki/File_system_permissions#Numeric_notation)

> btw: i just modified in my master branch, and pull, should i cancel this pull, make change in a new branch , and pull request again?

Please use ```git commit --amend``` to add changes to your commit. Use ```git push --force``` to overwrite your github repository/branch. The PR will afterwards update automatically. Please don't close this PR and open a new one.